### PR TITLE
Feature add alipay xborder

### PR DESCRIPTION
**Pull request by MarkusHobisch**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -358,6 +366,10 @@ The following payment methods are currently supported:
 * Paysafecard
 * Przelewy24
 * EPS
+* Alipay Cross-border
+* Paysafecard
```
Comment (horfic) [inline README.md]: check this list for duplicated, i see already duplicated paysafecard, p24, eps, etc.

File: README.md [hunk 2]
```diff
@@ -372,6 +384,8 @@ Additional fields `order-number`, `creditor-id` and `country` are required.
 | creditor-id | The hobex AG creditor ID (always "AT94ZZZ00000001251"). |
 | country | A 2-letter code, representing the first two characters of the consumer's IBAN. |
 
+You will find examples for each payment method in this demo project.  
+Furthermore you can register SEPA Direct Debit payments via the Austrian acquirer Hobex. If you use Hobex you have to choose `hobex-vt` as payment method instead of `sepadirectdebit`.
```
Comment (horfic) [inline README.md]: text looks to be duplicated

File: example-requests-embedded/alipay_cross_border_payment.json [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+    "request-id": "",
+    "requested-amount": {
+      "value": "1.01",
+      "currency": "EUR"
+    },
+    "account-holder": {
+      "first-name": "John",
+      "last-name": "Dao"
+    },
+    "transaction-type": "debit",
```
Comment (horfic) [inline example-requests-embedded/alipay_cross_border_payment.json]: is this correct? i do not see any payment method specified?

File: src/operations/operations.html [hunk 4]
```diff
@@ -260,11 +260,43 @@ <h3>Cancel: Refund a Przelewy24 payment</h3>
...
-
+    <div class="row">
+        <div class="col-xs-12">
+            <a class="btn btn-primary center-block text-center" href="../../index.html"
+               role="button"
+               style="text-align: right;float: right; margin: 30px 0 30px 0">Back</a>
+        </div>
+    </div>
 </div>
-<a href="../../index.html" style="text-align: right;float: right; margin: 30px">back</a>
```
Comment (horfic) [inline src/operations/operations.html]: is the back button now intentially removed?

File: src/register/embedded_overview.html [hunk 5]
```diff
@@ -191,13 +191,39 @@ <h2>EPS</h2>
...
+
+    <div class="row">
+        <div class="col-xs-12">
+            <a class="btn btn-primary center-block text-center" href="../../index.html"
+               role="button"
+               style="text-align: right;float: right; margin: 30px 0 30px 0">Back</a>
+        </div>
+    </div>
 </div>
-<a href="../../index.html" style="text-align: right;float: right; margin: 30px">back</a>
```
Comment (horfic) [inline src/register/embedded_overview.html]: is the back button now intentially removed?

Comment (MarkusHobisch) [reply src/register/embedded_overview.html]: It is replaced by an improved button layout.

**Pull request merged**